src/test/ui/parser/public-instead-of-pub.rs

```diff
@@ -0,0 +1,8 @@
+// Checks what happens when `public` is used instead of the correct, `pub`
+// edition:2018
+// run-rustfix
+public struct X;
+//~^ ERROR 4:8: 4:14: expected one of `!` or `::`, found keyword `struct`
```

Comment:
These line and column numbers still aren't necessary for the test to pass and just make the test output more brittle to changes.

Reply:
This is only for the ERROR right? Removing the line numbers for the HELP message borks the test, goes fine when only removing from the ERROR

Reply:
It should work for `HELP` too, if you grep for `HELP` in `src/test/ui` then there are any examples that do not have line/column numbers.

Reply:
Done, thanks!